src/components/Card.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getCountryData } from '../redux/detail/detail';
+
```

Comment:
Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [ ] _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [ ] OPTIONAL
The _code_ aspects in each procedure can be quoted with **backticks** to make the code description look cleaner.
- [ ] OPTIONAL
Kindly ensure that your ReadMe is _typo-free_. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)